### PR TITLE
Merge 'covid19biobank' column into 'capabilities'

### PR DESCRIPTION
**Pull request by tommydeboer**
#### Checklist
- [x] Functionality works & meets specifications
- [x] Code reviewed
- [x] Code unit/system tested
- User documentation updated
- [x] Clean commits
- [x] Updated CHANGELOG.md

### CODE REVIEW

**Comment by tommydeboer** [comment]
[[Biobank Services] Merge covid19biobank and capabilities in publisher](https://trello.com/c/2Z8bfM3L/879-biobank-services-merge-covid19biobank-and-capabilities-in-publisher)

File: src/molgenis/bbmri_eric/bbmri_client.py [hunk 1]
```diff
@@ -356,10 +356,10 @@ def import_as_csv(self, data: EricData):
     def _create_emx_archive(cls, data: EricData, directory: str) -> Path:
         archive_name = f"{directory}/archive.zip"
         with ZipFile(archive_name, "w") as archive:
-            for table_type in TableType.get_import_order():
-                file_name = f"{data.table_by_type[table_type].full_name}.csv"
+            for table in data.import_order:
```
Comment (jelmerveen) [inline src/molgenis/bbmri_eric/bbmri_client.py:359]: nice! Looks cleaner :)

File: src/molgenis/bbmri_eric/transformer.py [hunk 2]
```diff
@@ -141,3 +143,29 @@ def _set_combined_networks(self):
...
+                    f"Biobank {biobank['id']} uses deprecated '{covid}' column. "
+                    f"Use '{caps}' instead."
+                )
+                self.printer.print_warning(warning, indent=1)
+                self.warnings.append(warning)
+
+                if not biobank[caps]:
+                    biobank[caps] = []
+
+                biobank[caps].extend(biobank[covid])
```
Comment (jelmerveen) [inline src/molgenis/bbmri_eric/transformer.py:169]: What happens on duplicate entry?

Comment (tommydeboer) [reply src/molgenis/bbmri_eric/transformer.py:169]: I checked and there are no duplicate values between the `covid10biobanks` and `capabilities`

**Review by jelmerveen — APPROVED** [review]

**Pull request merged**